swarmrl/tasks/object_movement/rod_rotation.py

```diff
@@ -85,17 +85,25 @@ def _compute_angular_velocity(self, new_director: np.ndarray):
                 Angular velocity of the rod
         """
         # Compute the angular velocity
-        dot_product = np.clip(
-            np.dot(self._historic_rod_director, new_director), -1.0, 1.0
+
+        # angle between the two directors
+        angular_velocity = np.arctan2(
+            self._historic_rod_director[0] * new_director[1]
```

Comment:
also this angular velocity is more of a \Delta\Theta since you do no divide by a time. Is this done on purpose?

Reply:
Yeah my thought is in general that the delta velocity should be increasing and time in this case is between time slices which means it would be unnecessary information. Therefore it is just a delta but I can re-write it like that.

Reply:
To your first point yes and it should be, I will change that

Reply:
I have made both changes.